packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec

```diff
@@ -24,13 +24,13 @@ Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.10
-Requires: %{?scl_prefix}rubygem(foreman-tasks) < 1
+Requires: %{?scl_prefix}rubygem(foreman-tasks) < 2
```

Comment:
this doesn't match the gemspec - https://github.com/ATIX-AG/foreman_scc_manager/blob/master/foreman_scc_manager.gemspec#L25 
we should drop the upper limit